docs/database/cosmosdb.md

```diff
@@ -90,4 +90,20 @@ In addition to Cosmos DB attributes, all spans include
 | `db.cosmosdb.sub_status_code`        | `0` |
 | `db.cosmosdb.request_charge`         | `7.43` |
 
+## Network-level attributes
+
+<!-- semconv db.cosmosdb(full,tag=network-level-tech-specific) -->
```

Comment:
I suggest to put everything rntbd-related into rpc folder and reference it from here.